lib/starlark_compiler/ast.rb

```diff
@@ -72,6 +72,8 @@ def node(obj)
           False.new
         when Numeric
           Number.new(obj)
+        when Variable
```

Comment:
this shouldnt be necessary -- this is for mapping ruby stdlib types into ast nodes.